docs/examples/encrypt_and_transfer.py

```diff
@@ -0,0 +1,70 @@
+from gladier import GladierBaseClient, generate_flow_definition, GladierBaseTool
+from pprint import pprint
+import pathlib
+
+
+def make_files(**data):
+    import os
+
+    input_path = pathlib.Path(data['make_input'])
+
+    if '~' in input_path:
+        input_path = os.path.expanduser(input_path)
+
+    if not os.path.exists(input_path):
+        os.makedirs(input_path)
```

Comment:
`pathlib` isn't imported here. Also, we shouldn't mix `os.path` and `pathlib`, but rather switch to one or the other for manipulating paths. Pathlib has methods for expanding `~` and mkdir.

Reply:
Changed it, thank you!

Reply:
As a future reference. Lets keep `os.path` as standard for now and wait for python 4.x definitions on it. Then we can make a conscious decision to move to `Path` or not.